src/JWT/JWT.csproj

```diff
@@ -3,10 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>net35;netstandard1.3</TargetFrameworks>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
-    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
-    <DefineConstants>NET35</DefineConstants>
-    <FrameworkPathOverride>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">

```

Comment:
you can just add net462 like:
`<TargetFrameworks>net35;net462;netstandard1.3</TargetFrameworks>`.

But this is not needed because the net35 can be used by all frameworks.